Frontend/ui-customer/src/Screens/CustomerUtils/MyProfileScreen/Controller.purs

```diff
@@ -234,6 +235,10 @@ updateProfile (GetProfileRes profile) state = do
                     Just "" -> ""
                     Just name -> (" " <> name)
                     Nothing -> ""
+      mobileNumber = case profile.maskedMobileNumber of
+                      Just "" -> ""
+                      Just mn -> mn
+                      Nothing -> ""
```

Comment:
`  mobileNumber = case profile.maskedMobileNumber of
                      Just mn -> mn
                      _ -> ""`
                      
or we can directly write `fromMaybe "" profile.maskedMobileNumber`

ye starting m not sure kisne ese likha hua 😢

Reply:
i just copy pasted yesterday night 🤣